packages/lab/src/pill-next/PillNext.tsx

```diff
@@ -1,22 +1,15 @@
-import {
-  forwardRef,
-  MouseEvent,
-  KeyboardEvent,
-  ComponentPropsWithoutRef,
-} from "react";
+import { forwardRef, ComponentPropsWithoutRef } from "react";
 import clsx from "clsx";
 import { useWindow } from "@salt-ds/window";
 import { useComponentCssInjection } from "@salt-ds/styles";
 import { makePrefixer, useButton } from "@salt-ds/core";
 import pillCss from "./PillNext.css";
-
-export type PillClickEvent =
-  | MouseEvent<Element, globalThis.MouseEvent>
-  | KeyboardEvent<HTMLDivElement>;
+import { PillNextCloseButton } from "./PillNextCloseButton";
 
 export interface PillNextProps extends ComponentPropsWithoutRef<"button"> {
   /* If true the pill will be disabled */
   disabled?: boolean;
+  onClose?: () => void;
```

Comment:
This type is incorrect after the update. I think it would be better to pick the type from PillNextCloseButton